### PR TITLE
docs: add public roadmap

### PR DESCRIPTION
**Pull request by jlipps**

### CODE REVIEW

**Comment by jlipps** [comment]
cc @eglitise

**Comment by eglitise** [comment]
Seems like most of the items are for the Inspector 😄 but just to clarify, saved session export/import already exists, but I think it only works on one session at a time. The end goal is to extend this to simple bulk import/export procedures.

**Review by KazuCocoa — APPROVED** [review]
lgtm overall

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+This is not a traditional roadmap. We don't have a typical product team with a typical product
+manager. The fact that the items below exist on a roadmap is no indication that we will ever have
+the opportunity to work on them. Instead this is more like a wishlist of outcomes that the project
+thinks would be valuable. If and when we get additional contributions of development work, this is
+a set of items that would be great for someone to work on! Ideally, interested devs would find
+something on this list and own it from start to finish! Let anyone on the project know if you're
+interested and we can help get you going.
+
+- **WebDriver BiDi support in the Appium server**: the WebDriver BiDi spec is underway, which details a different method for communicating with drivers. We should implement handling for these methods in our server so that Appium drivers can take advantage of the BiDi spec without additional work on their part.
```
Comment (mykola-mokhnach) [inline ROADMAP.md:11]: would it be more logical to split these points into groups? For example I can already see the biggest group related to the Appium Inspector

Comment (jlipps) [reply ROADMAP.md:12]: good idea, added sections

**Review by mykola-mokhnach — APPROVED** [review]

**Pull request merged**